ServerRecording/README.MD

```diff
@@ -0,0 +1,66 @@
+---
+page_type: sample
+languages:
+- C#
+products:
+- azure
+- azure-communication-services
+---
+
+[![Deploy to Azure](https://aka.ms/deploytoazurebutton)](https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure-Samples%2Fcommunication-services-web-calling-hero%2Fmain%2Fdeploy%2Fazuredeploy.json)
```

Comment:
I clicked this button, and I'm still not sure what it's intended purpose is, or at least with how it relates to this sample or call recording in general. 

If you do decide to include it, I would give a written description above it detailing what the expected outcome is.

Reply:
Removed it, not required.